app/templates/index.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui container">
```

Comment:
Add `main-heading` class of `Call For Speakers` also.

Reply:
Done.